supported_formats_and_operations.md

```diff
@@ -43,6 +43,12 @@ The supported preprocessing operations.
     - `min_percentile` the lower percentile used for normalization, in range 0 to 100.
     - `max_percentile` the upper percentile used for normalization, in range 0 to 100. Has to be bigger than `upper_percentile`.
```

Comment:
```suggestion
    - `max_percentile` the upper percentile used for normalization, in range (1, 100]. Has to be bigger than `upper_percentile`. Note: We do not allow values <=1 to avoid mistakenly specifying the percentiles in [0,1]
```